public/components/explorer/home.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { EuiBasicTable } from '@elastic/eui';
+import { Search } from '../common/seach/search';
+
+export const Home = (props: any) => {
+
+  const history = useHistory();
+
+  const histories = [{
+    historiy: 'sample histories list, all histories will be listed here, showing the last N queries that user has executed'
+  }];
+
+  const column = [{
+    field: 'historiy',
+    name: 'History'
+  }];
+
+  const actionItems = [
+    {
+      text: 'Run',
+      iconType: '',
+      attributes: {
+        fill: 'fill'
+      },
+      handlers: {
+        onClick: () => {
+          history.push('/event/explorer');
+        }
+      }
+    },
+    {
+      text: 'Live',
+      iconType: 'play',
+      handlers: {
+        onClick: () => {
+          console.log('refresh clicked');
+        }
+      }
+    }
+  ];
+
+  return (
+    <div className="dscAppContainer">
+      <h1>Event Analytics</h1>
+      <Search 
+        query={""}
+        handleQueryChange={ () => {} }
+        handleQuerySearch={ () => {} }
+        startTime={ 'now-15m' }
+        endTime={ 'now' }
+        setStartTime={ () => {} }
+        setEndTime={ () => {} }
+        setIsOutputStale={ () => {} }
+        liveStreamChecked={ false }
+        onLiveStreamChange={ () => {} }
+        actionItems={ actionItems }
+      />
+      {/* <Search 
```

Comment:
Same question on this commented-out portion

Reply:
There will be a next to-do item on this, currently in the middle of the code changes. Shall I keep this or have to finish the entire thing before merging?